server/app/repository.py

```diff
@@ -148,6 +148,14 @@ def _cursor(self) -> Cursor:
         context = self._transaction_context
         return context.get("cursor")
 
+    @use_cursor
+    def count_observations(self) -> int:
+        q = ("SELECT COUNT(*) as count "
```

Comment:
Is your terminal 40 chars wide? This kind or line wrap looks horrible.

Reply:
You mentioned PEP8 saying line wrap at 80 chars. Are we moving to PEP4 now? :rofl: